src/App.js

```diff
@@ -165,14 +166,16 @@ const movieDescriptionStyles = {
 }
 
 const Header = () => {
-  const { wrapper, slot } = headerStyles;
+  const { wrapper, leftSlot, leftSide, rightSide } = headerStyles;
   return (
     <div style={wrapper}>
-      <div style={slot}><p>SOBRE</p></div>
-      <div style={slot}><p>CONTATO</p></div>
-      <div style={slot}>
-        <div style={{flex:1, textAlign: 'end', marginRight: 5}}><p>🇧🇷</p></div>
-        <div style={{flex:1}}><p>🇺🇸</p></div>
+      <div style={leftSide}>
+        <div style={leftSlot}><p>SOBRE</p></div>
+        <div style={leftSlot}><p>CONTATO</p></div>
+      </div>
+      <div style={rightSide}>
+        <div><p>🇧🇷</p></div>
+        <div><p>🇺🇸</p></div>
```

Comment:
vai ter em duas línguas? 🤔

Reply:
um dia.. hahah

mas da pra tirar isso dai enquanto não tem, eu to deixando mais pra me cobrar de fazer hahah

Reply:
ehehehe